Cargo.lock

```diff
@@ -2110,6 +2160,39 @@ dependencies = [
  "serde_yaml",
 ]
 
+[[package]]
+name = "openssl"
```

Comment:
we will have problems on some architecture with this

Reply:
good point, tungstenite is only used in the tests, so I think turning off the tls feature may be fine.